flit/validate.py

```diff
@@ -125,17 +127,19 @@ def _is_identifier_attr(s):
                                 '{} = {}'.format(groupname, k, v))
     return problems
 
+
 # Distribution name, not quite the same as a Python identifier
 NAME = re.compile(r'^([A-Z0-9]|[A-Z0-9][A-Z0-9._-]*[A-Z0-9])$', re.IGNORECASE)
 r''
 VERSION_SPEC = re.compile(r'(~=|===?|!=|<=?|>=?)\s*[A-Z0-9\-_.*+!]+$', re.IGNORECASE)
 REQUIREMENT = re.compile(NAME.pattern[:-1] +  # Trim '$'
-     r"""\s*(?P<extras>\[.*\])?
+                         r"""\s*(?P<extras>\[.*\])?
          \s*(?P<version>[(=~<>!][^;]*)?
          \s*(?P<envmark>;.*)?
      $""", re.IGNORECASE | re.VERBOSE)
 MARKER_OP = re.compile(r'(~=|===?|!=|<=?|>=?|\s+in\s+|\s+not in\s+)')
 
+
```

Comment:
Not a big deal, but as a general rule, please try not to reformat code beyond the bit that you're working on.